src/v2/view-builder/views/consent/EnduserEmailConsentView.js

```diff
@@ -0,0 +1,91 @@
+import { loc } from 'okta';
+import { BaseForm, BaseHeader } from '../../internals';
+import ConsentViewForm from './ConsentViewForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import HeaderBeacon from '../../components/HeaderBeacon';
+import { getIconClassNameForBeacon } from '../../utils/AuthenticatorUtil';
+import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
+
+export default BaseAuthenticatorView.extend({
+  className: 'enduser-email-consent',
+
+  Header: BaseHeader.extend({
+    HeaderBeacon: HeaderBeacon.extend({
+      getBeaconClassName: function() {
+        return getIconClassNameForBeacon(AUTHENTICATOR_KEY.EMAIL);
+      },
+    }),
+  }),
+
+  postRender() {
+    // Move buttons in DOM to match visual hierarchy to fix tab order.
+    // TODO https://oktainc.atlassian.net/browse/OKTA-350521
+    const buttonContainer = this.$el.find('.o-form-button-bar');
+    buttonContainer.find('.button-primary').appendTo(buttonContainer);
+    buttonContainer.find('.button-primary').removeClass('button-primary');
+  },
+
+  Body: ConsentViewForm.extend({
+    title() {
+      return loc('oie.consent.enduser.title', 'login');
+    },
+    save() {
+      return loc('oie.consent.enduser.accept.label', 'login');
+    },
+    cancel() {
+      return loc('oie.consent.enduser.deny.label', 'login');
+    },
+    initialize() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      if (this.model.get('browser')) {
```

Comment:
nit:
are you able to use handlebar template?
```
tpl({{if browser}}...{{if app}...)(this.model.pick('browser', 'app'))
```

Reply:
There seems to be a lot of restrictions with our `lint`. Looks like the only way I am able to do this is -
```
// One looong line - so I'll need an eslint directive to ignore long lines here.
// note- i can not do string concatenation for some odd reason??
// neither can i use backticks here. it fails.
const getInfo = hbs('{{#if browser}}<div class="enduser-email-consent--info"><i class="enduser-email-consent--icon browser-icon"></i><div>{{browser}}</div></div>{{/if}}{{#if app}}<div class="enduser-email-consent--info"><i class="enduser-email-consent--icon app-icon"></i><div>{{app}}</div></div>{{/if}}');
.
.
.
const info = getInfo(this.model.pick('browser', 'app'));
this.add(info);
```

It makes the `initialize` function definition looks nice and tidy. What do you think?

Reply:
Create another View? 
or
```
 hbs`
...
`
```

Try to search usage of `hbs` in `src`